lib/matplotlib/artist.py

```diff
@@ -757,15 +757,16 @@ def set_clip_box(self, clipbox):
 
         Parameters
         ----------
-        clipbox : `.BboxBase` or None
-            Typically would be created from a `.TransformedBbox`. For
-            instance ``TransformedBbox(Bbox([[0, 0], [1, 1]]), ax.transAxes)``
-            is the default clipping for an artist added to an Axes.
+        clipbox : `~matplotlib.transforms.Bbox` or None
```

Comment:
```suggestion
        clipbox : `~matplotlib.transforms.BboxBase` or None
```

`TransformedBbox` referenced in the first line is a sub-class of `BBoxBase` not `BBox`.